common/test/Configuration.Provisioning.cs

```diff
@@ -19,9 +19,6 @@ public static partial class Provisioning
 
             public static string IdScope => GetValue("DPS_IDSCOPE");
 
-            public static string TpmDeviceRegistrationId => GetValue("DPS_TPM_REGISTRATIONID");
-
-            public static string TpmDeviceId => GetValue("DPS_TPM_DEVICEID");
 
```

Comment:
registration id's are now generated on the fly, not hardcoded to the DPS instance. So I removed these fields